{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -58,14 +57,9 @@ deps =
     -r{toxinidir}/requirements/test.txt
 commands =
     touch tests/__init__.py
-    pylint {{ cookiecutter.app_name }}
-    pylint tests
-    pylint test_utils
-    pylint --py3k {{ cookiecutter.app_name }}
-    pylint --py3k tests
-    pylint --py3k test_utils
+    pylint {{ cookiecutter.app_name }} tests test_utils
+    pylint --py3k {{ cookiecutter.app_name }} tests test_utils
     rm tests/__init__.py
     pycodestyle {{ cookiecutter.app_name }} tests
     pydocstyle {{ cookiecutter.app_name }} tests
     isort --check-only --recursive tests test_utils {{ cookiecutter.app_name }} manage.py setup.py test_settings.py
-    make help
```

Comment:
This is actually here to verify that the Makefile is formatted correctly; it throws an error and fails the test if spaces were used instead of tabs, etc.